yellowbrick/datasaurus.py

```diff
@@ -29,222 +29,1175 @@
-         77.95443505, 77.08333777, 76.06355025, 77.68201632, 76.87808198,
-         76.94850272, 77.86405471, 75.77145009, 52.33156913, 52.59281837,
-         50.47704772, 75.29647509, 75.57395413, 75.40052716, 75.87099084,
-         75.60588476, 75.89557705, 75.7465632 , 75.14234148, 50.66177956,
-         50.69985064, 50.91894087, 50.72525854, 51.26387123, 51.25091965,
-         50.78515721, 50.50139658, 50.73367454, 50.71137854, 50.8127449 ,
-         51.01423295, 50.35352141, 50.43552957, 50.63098196, 51.0668072 ,
-         50.79235473, 50.55127806, 50.55975806, 75.32597855, 75.04472578,
-         75.28708772, 75.23996998, 75.1524592 , 75.96184009, 75.44806251,
-         75.75938382, 50.3782623 , 50.53363501, 77.50090732, 50.69112419,
-         49.99039495, 50.12718203],
-        [90.86741233, 89.10239459, 85.4600474 , 83.05766953, 82.93782178,
-         82.97525357, 82.91489113, 82.92908498, 82.8742005 , 82.92409777,
-         82.82118411, 51.48738653, 51.41484656, 52.07679944, 51.71207905,
-         50.70890793, 51.65304675, 51.18198917, 51.41855226, 52.12301105,
-         50.62155476, 50.07473901, 51.5024421 , 51.86195209, 52.25779061,
-         51.19794432, 82.94182882, 83.75234297, 51.97525067, 51.07339565,
-         51.3380902 , 52.1768375 , 51.20176505, 50.44143545, 51.41620515,
-         17.14563109, 17.14132373, 17.08190869, 16.92501353, 50.66196341,
-         51.39909748, 50.79528152, 50.68603709, 51.52476126, 17.40539097,
-         17.20372213, 17.09382391, 17.11384266, 17.02374454, 17.11492526,
-         17.07777732, 16.98102188, 17.03857897, 50.69056272, 51.29446922,
-         51.59435617, 52.33576553, 52.04552865, 51.74673004, 50.31866042,
-         51.46182482, 52.12368985, 51.9671367 , 82.98566202, 83.11447934,
-         82.98265686, 82.84604113, 83.18462233, 82.90990147, 82.93532841,
-         83.96992038, 82.99366549, 83.09951912, 83.7083177 , 82.9019501 ,
-         51.43887623, 51.30411215, 51.59365408, 94.24932783, 92.97911753,
-         88.38644174, 83.90349738, 83.46230334, 82.91945886, 82.88405139,
-         82.93211578, 82.96238879, 83.03499717, 82.9452793 , 51.15177033,
-         50.47557897, 52.15779927, 52.10465206, 51.16563781, 51.8675623 ,
-         51.90751654, 49.66254553, 17.11125121, 51.87886035, 51.39159152,
-         17.04828941, 17.01565319, 17.06219214, 17.04110689, 17.13489391,
-         17.06772306, 17.16994971, 17.10571651, 16.75492389, 17.07814052,
-         17.08518438, 17.14760476, 16.90746981, 17.16234971, 17.24045586,
-         17.18019648, 17.10577072, 16.99296341, 17.08831585, 16.57271805,
-         17.22109553, 17.06474308, 17.0651685 , 17.07652235, 17.20885971,
-         17.20421434, 17.08465518, 17.09388377, 15.77189199, 17.00426226,
-         16.17493491, 17.03184749, 17.0049424 , 16.69484223, 17.04514941,
-         16.94292965, 16.94627981, 17.01958137, 50.16698595, 87.51396042,
-         83.99735692, 82.99075   ]])]
+    np.array(
```

Comment:
Ouch ... this is also not fun. I wonder if we should just put anscombe and datasaurus into `datasets`?

Reply:
Yeah sorry 🤕